depthai_nodes/ml/parsers/yolo.py

```diff
@@ -125,22 +133,35 @@ def build(
 
         self.output_layer_names = bbox_layer_names + kps_layer_names + masks_layer_names
 
-        metadata = head_config["metadata"]
-        self.conf_threshold = metadata["conf_threshold"]
-        self.n_classes = metadata["n_classes"]
-        self.iou_threshold = metadata["iou_threshold"]
-        self.anchors = metadata["anchors"]
-        if "mask_conf" in metadata:
-            self.mask_conf = metadata["mask_conf"]
-        if "n_keypoints" in metadata:
-            self.n_keypoints = metadata["n_keypoints"]
-        if "yolo_version" in metadata:
-            try:
-                self.yolo_version = YOLOVersion(metadata["yolo_version"].lower())
-            except ValueError as err:
-                raise ValueError(
-                    f"Invalid YOLO version {metadata['yolo_version']}. Supported YOLO versions are {[e.value for e in YOLOVersion][:-1]}."
-                ) from err
+        # metadata = head_config["metadata"]
+        self.conf_threshold = head_config.get("conf_threshold", self.conf_threshold)
+        self.n_classes = head_config.get("n_classes", self.n_classes)
+        self.iou_threshold = head_config.get("iou_threshold", self.iou_threshold)
+        self.mask_conf = head_config.get("mask_conf", self.mask_conf)
+        self.anchors = head_config.get("anchors", self.anchors)
+        self.n_keypoints = head_config.get("n_keypoints", self.n_keypoints)
+        yolo_version = head_config.get("subtype", self.yolo_version)
```

Comment:
I would rather go then with `subtype` instead of `yolo_version` on all names (attributes, variables, parameters, etc.) to be consistent. And Enum could be `YOLOSubtype` instead of `YOLOVersion`?

Reply:
Changed all names to `subtype` and `YOLOSubtype` in [009a00b](https://github.com/luxonis/depthai-nodes/pull/109/commits/009a00b8745a2cb13180796441ce3195d323e673).